CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-* None
+* `New-TssSession` - added -UseDefaultCredentials parameter
```

Comment:
You will need to rebase your branch to pick up the previous merged PR.

As well, this entry should go under the `Added` section.

Reply:
done